unshare/unshare.go

```diff
@@ -84,11 +84,6 @@ func (c *Cmd) Start() error {
 		c.Env = append(c.Env, fmt.Sprintf("_Buildah-ctty=%d", len(c.ExtraFiles)+3))
 		c.ExtraFiles = append(c.ExtraFiles, c.Ctty)
 	}
-	if c.GidMappingsEnableSetgroups {
```

Comment:
This nullifies any effect that setting the field has, but leaves the field in the structure.  Why?

Reply:
it is still used to write `allow` or `deny` to the `/proc/PID/setgroups` file